scripts/game_structure/game/settings/settings.py

```diff
@@ -0,0 +1,99 @@
+import os
+import traceback
+from typing import Any, Tuple
+
+import ujson
+
+from scripts.game_structure.game.save_load import safe_save
+from scripts.housekeeping.datadir import get_save_dir
+
+settings_changed: bool = False
+settings = {"moon_and_seasons_open": False}
+setting_lists = {}
+
+
+def save_game_settings(currentscreen=None):
+    """Save user settings for later use"""
+    if os.path.exists(get_save_dir() + "/settings.txt"):
+        os.remove(get_save_dir() + "/settings.txt")
+    global settings_changed
+
+    settings_changed = False
+    try:
+        safe_save(get_save_dir() + "/settings.json", settings)
```

Comment:
Moving forward, we should try to use Pathlib instead of constructing path literals as strings.